docker-compose.yml

```diff
@@ -129,5 +129,5 @@ volumes:
   grafana_data:
   influxdb_data:
   shared:
-  op_geth:
```

Comment:
I see why we remove `op_geth`, but adding `eigenda-data` doesn't belong to this PR, right?

Reply:
Good point, removed